my_compassion/templates/my2_login_template.xml

```diff
@@ -0,0 +1,106 @@
+        >
+                    <div class="my2-login-box text-center">
+                        <img
+              src="/my_compassion/static/src/img/compassion-icon.png"
+              class="icon mb-3"
+              alt="MyCompassion Icon"
+            />
+                        <h2 class="welcome">Welcome to<br />MyCompassion</h2>
+                        <form
+              t-attf-action="/web/login{{ '?redirect=' + redirect if redirect else '' }}"
+              method="post"
+              class="my2-login-form"
+            >
+                            <input
+                type="hidden"
+                name="csrf_token"
+                t-att-value="request.csrf_token()"
+              />
+
+                            <div class="form-group">
+                                <input
+                  type="text"
+                  name="login"
+                  class="form-control"
+                  placeholder="Email Address"
+                  required="required"
+                />
+                            </div>
+                            <div class="form-group position-relative">
+                                <input
+                  type="password"
+                  id="password"
+                  name="password"
+                  class="form-control"
+                  placeholder="Password"
+                  required="required"
+                />
+                                <i
+                  class="fa fa-eye-slash toggle-password"
+                  id="eye_password"
+                />
+                            </div>
+
+                            <a
+                href="/web/reset_password"
+                class="forgot-password d-block mb-2 text-start"
+              >Forgot password?</a>
+
+                            <button
+                type="submit"
+                class="btn btn-primary btn-block"
+              >Login</button>
+
+                            <div class="mt-3">
+                                <p>Don't have an account?</p>
+                                <a
+                  href="/web/signup?redirect=/my/home"
+                >Create one now</a>
+                            </div>
+                            <div class="mt-2">
+                                <p>Need help?</p>
+                                <a href="/contactus">Contact us</a>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+            </t>
+        </xpath>
+    </template>
+
+    <!-- Add custom CSS to frontend -->
+    <template id="my2_login_styles" inherit_id="website.assets_frontend">
+        <xpath expr="." position="inside">
+            <link
+        rel="stylesheet"
+        href="/my_compassion/static/src/css/my2_login.css"
+      />
+            <link
+        rel="stylesheet"
+        href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      />
```

Comment:
We should not import external fonts/icons like this, when there is the need, they should be imported in /static.
```suggestion
```